src/features/rewards/tableAdsHistory/index.tsx

```diff
@@ -0,0 +1,245 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import {
+  StyledTHFirst,
+  StyledTHOther,
+  StyledTHLast,
+  StyledAdContentDiv,
+  StyledLogo,
+  StyledLogoDiv,
+  StyledAdInfoDiv,
+  StyledAdBrand,
+  StyledAdInfo,
+  StyledAdStatDiv,
+  StyledAdStat,
+  StyledAdStatActions,
+  StyledCategoryContentDiv,
+  StyledCategoryName,
+  StyledCategoryActions,
+  StyledAdLink,
+  StyledNoAdHistoryDiv,
+  StyledNoLogoDiv,
+  StyledTD
+} from './style'
+import { Row } from '../../../components/dataTables/table'
+import { StyledTable } from '../../../components/dataTables/table/style'
+import ThumbLikePicker from '../thumbLikePicker'
+import CategoryLikePicker from '../categoryLikePicker'
+import DropMenu from '../dropMenu'
+import AdRowsDetails from '../adRowsDetails'
+import { getLocale } from '../../../helpers'
+
+export interface DetailRow {
+  [key: string]: any
+  id: string
+  date: string
+  adDetailRows: AdDetailRow[]
+}
+
+interface AdContent {
+  brand: string
+  brandInfo: string
+  brandLogo: string
+  brandUrl: string
+  brandDisplayUrl: string
+  likeAction: number
+  adAction: 'click' | 'dismiss' | 'view' | 'landed'
+  savedAd: boolean
+  flaggedAd: boolean
+  logoUrl?: string
+  onThumbUpPress?: () => void
+  onThumbDownPress?: () => void
+  onMenuSave?: () => void
+  onMenuFlag?: () => void
+}
+
+interface CategoryContent {
+  category: string
+  optAction: number
+  onOptInAction?: () => void
+  onOptOutAction?: () => void
+}
+
+export interface AdDetailRow {
+  id: string
+  adContent: AdContent
+  categoryContent: CategoryContent
+}
+
+export interface Props {
+  header: string[]
+  id?: string
+  testId?: string
+  children?: React.ReactNode
+  headerColor?: boolean
+  rows?: DetailRow[]
+  allItems?: number
```

Comment:
what this one means? if I understand correctly is what tab do we want to show?

Reply:
There are no tabs in this design.

This is used to display ad items according to the 'All' | 'Saved' | 'thumb-up' filter